Makefile

```diff
@@ -49,10 +49,11 @@ GPG_KEYID ?= asarai@suse.de
 # Some targets need cgo, which is disabled by default when cross compiling.
 # Enable cgo explicitly for those.
 # Both runc and libcontainer/integration need libcontainer/nsenter.
-runc static localunittest: export CGO_ENABLED=1
+#enable stack protection with CGO compile.
+runc static localunittest: export CGO_ENABLED=1 && CGO_CFLAGS='-fstack-protector-strong'
```

Comment:
1. What if someone sets CGO_CFLAGS? Maybe use `+=` instead?
2. Frankly, I don't understand the use of `&&` here. Note this is GNU make, not shell.

This all this, seems like it should be
```make
export CGO_ENABLED=1 CGO_CFLAGS+='-fstack-protector-strong'
```

Reply:
One other thing, what if a different C backend is used (not gcc) and it could not understand `-fstack-protector-strong`? Just thinking out loud here.